### DISCUSSION TITLE
Wormhole Limit Increase Proposal for Scroll

### DISCUSSION THREAD

**Discussion by 0x-beachball** [wormhole-foundation/wormhole#4006] [Governance]
Scroll and its underlying DeFi ecosystem are growing rapidly, especially since the launch of their Marks program. Scroll is a zkEVM L2 with significant interest from the community and major protocols. I am a core contributor at Synonym Finance, a Wormhole-based multi-chain money market with deployments on Ethereum, Arbitrum, Optimism, Base and crucially, Scroll. In the last 24 hours, Synonym alone has triggered multiple Governor holds on high value transactions that are limiting growth for our protocol, Scroll, and Wormhole's volume figures. We are proposing an increase in the per transaction limits, which are currently:

- 500K rolling daily limit
- 50K Big TX limit with hold for 24 hours

Synonym, which is still a growing protocol, has triggered both these limits alone in the past 12 hours. We are proposing an increase in limits to the sizes listed below. We do not believe this to be a dramatic expansion - this increase would simply cover the flow demand of smaller protocols working with LPs, whales and those looking to participate in the Wormhole ecosystem:

- 1.5M rolling daily limit
- 500K Big TX limit

The need for this increase is clear - the Scroll ecosystem has shown rapid growth and is closing in on $1B in aggregate value (note that DeFi Llama methodology reports $0.5B based on their TVL calculations which removes value based on lending). 

<img width="2038" alt="Screenshot 2024-07-02 at 3 56 54 PM" src="https://github.com/wormhole-foundation/wormhole/assets/139830188/9456e95a-d979-4b20-8253-9802ab392ec5">

Scroll has posted $10M inflow days since launching their Sessions campaign and we believe that those flows will continue to grow as more Marks are added across upcoming Sessions Epochs. We welcome feedback on this proposal and hope to see an end to this congestion soon!

**Comment by claudijd** [comment]
@0x-beachball, thanks for your proposal.  It's worth noting that the current daily limit consumption is 20% of the $500K limit (or $100K used).  There was also a new version of the Governor (https://x.com/wormhole/status/1806726933511606298) that has been released, so the inflows you mention above, if they make up of stables, will only help lower the limit consumption of the governor limits going forward.

Lastly, a proposal to increase the bigtx limit disproportionally to the daily limit is a bit out of the convention to what Guardians have supported historically, which is usually 10% of the daily limit, and you're proposing a 10x increase in the bigtx when usage is 5x below the actual daily limits.  Having a whale (which I assume to be rare) hit a bigtx limit occasionally seems acceptable to me, as it offers incident responders some time to intervene should the transaction be determined fraudulent.

**Comment by krzysztof-cywinski** [comment]
The problem with a relatively small TX limit for lending protocols is when users want to loop their position.
If a whale wants to deposit and leverage they usually will do a loop of:
- deposit X
- borrow Y
- swap Y -> X
- repeat

If the initial deposit is $200k they'll be hitting the TX limit twice per loop and setting up a position fully will take them at least a week.
This is both a terrible user experience and a huge risk, because the user would not be able to deleverage quickly if need be and will risk liquidation.
This is a dealbreaker for a lot of LPs.

**Comment by 0x-beachball** [comment]
@claudijd - That context on typical request sizes is helpful! Would be great to work through what an appropriate, near-term tx limit change would look like. As Krzysztof mentioned above, these limits do not just impact entry into new ecosystems via Wormhole, but present serious risks for any DeFi builders looking to take advantage of the infrastructure.
